services/QuillLMS/client/app/bundles/Teacher/components/classrooms/class_code_link.tsx

```diff
@@ -9,59 +9,32 @@ interface ClassCodeLinkProps {
   classroom: any;
 }
 
-interface ClassCodeLinkState {
-  showSnackbar: boolean
-}
-
-export default class ClassCodeLink extends React.Component<ClassCodeLinkProps, ClassCodeLinkState> {
-  constructor(props) {
-    super(props)
-
-    this.showSnackbar = this.showSnackbar.bind(this)
-  }
+export const ClassCodeLink = ({ next, back, showSnackbar, classroom }: ClassCodeLinkProps) => {
+  const link = `quill.org/join/${classroom?.code}`
```

Comment:
Is it possible for `classroom` to be null here? That seems like confusing UX since this link won't work.